backend/src/main/java/com/huellapositiva/domain/service/ProposalService.java

```diff
@@ -325,4 +329,41 @@ public void changeStatusToPublishedFromEnrollmentClosed(String proposalId) {
                 .name("PUBLISHED").build();
         jpaProposalRepository.updateStatusById(proposalId, jpaProposalStatus);
     }
+
+    @Scheduled(cron = "0 0 * * * * ")
```

Comment:
Trim expression

Reply:
Not sure I understand, tried with `@daily` but didn't like it.

Reply:
Trim means removing leading and trailing spaces so `0 0 * * * * ` becomes `0 0 * * * *`